app/controllers/redirects_controller.rb

```diff
@@ -0,0 +1,30 @@
+class RedirectsController < Comfy::Admin::Cms::BaseController
+  def new
+    @redirect = Redirect.new
+  end
+
+  def create
+    @redirect = Redirect.new(redirect_params)
+
+    if @redirect.save
+      flash[:success] = 'Successfully created redirect'
+      redirect_to redirects_path
+    else
+      render :new
+    end
+  end
+
+  def index
+    @redirects = Redirect.all
+  end
+
+  private
+
+  def load_admin_site
+    @site = Comfy::Cms::Site.first
+  end
```

Comment:
I've now removed this. Comfy overrides render. As we're inheriting from `Comfy::Admin::Cms::BaseController`, there was code using `site` if the template could not be found. now that the templates exist (which they didn't before the tests were create) this is no longer needed